src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx

```diff
@@ -0,0 +1,220 @@
+
+In this guide, you will learn how to create python and golang functions in Amplify Functions.
+
+# Python functions
+To get started, create a new directory and a resource file, `amplify/functions/say-hello/resource.ts`. Then, define the Function with `defineFunction`:
+
+```ts title="amplify/functions/say-hello/resource.ts"
+import { defineFunction } from "@aws-amplify/backend";
+import { DockerImage, Duration } from "aws-cdk-lib";
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+import { execSync } from "child_process";
+import * as path from "path";
+
+export const sayHello = defineFunction(
+  (scope) =>
+    new Function(scope, "say-hello", {
+      handler: "index.handler",
+      runtime: Runtime.PYTHON_3_9, // or any other python version
+      timeout: Duration.seconds(20), //  default is 3 seconds
+      code: Code.fromAsset(functionDir, {
+        bundling: {
+          image: DockerImage.fromRegistry("dummy"),
+          local: {
+            tryBundle(outputDir: string) {
+              execSync(
+                `python3 -m pip install -r ${path.join(functionDir, "requirements.txt")} -t ${path.join(outputDir)} --platform manylinux2014_x86_64 --only-binary=:all:`
+              );
+              execSync(`rsync -rLv ${functionDir}/* ${path.join(outputDir)}`);
+              return true;
+            },
+          },
+        },
+      }),
+    })
+);
+```
+
+Next, create the corresponding handler file at `amplify/functions/say-hello/handler.ts`. This is where your function code will go.
+
+```ts title="amplify/functions/say-hello/index.py"
+import json
+        
+def handler(event, context):
+  return {
+      "statusCode": 200,
+      "body": json.dumps({
+          "message": "Hello World",
+      }),
+  }
+```
+
+The handler file _must_ export a function named "handler". This is the entry point to your function. For more information on writing functions, refer to the [AWS documentation for Lambda function handlers using python](https://docs.aws.amazon.com/lambda/latest/dg/python-handler.html).
+
+If you need python packages, you can add them to a `requirements.txt` file in the same directory as your handler file. The `bundling` option in the `Code.fromAsset` method will install these packages for you.
+Create a `requirements.txt` file in the same directory as your handler file. This file should contain the names of the packages you want to install. For example:
+
+```txt title="amplify/functions/say-hello/requirements.txt"
+request==2.25.1
+some-other-package>=1.0.0
+```
+
+You're now ready to deploy your python function. Next is the same process as the Node.js/TypeScript function. Go to [Common steps for all languages](#common-steps-for-all-languages) to continue.
+
+# Go functions
+To get started, install go package from npm. go to your backend directory, you should see amplify folder inside it. and run the following command:
+```npm i @aws-cdk/aws-lambda-go-alpha```
```

Comment:
We should refrain from using alpha packages in docs.

We should:
- either convert this sample to use `Code.from..` and plain `Function` (like in case of python).
- or just remove Go sample.

Reply:
Here is an example of how this can be implemented using `Function`:

```
import { defineFunction } from "@aws-amplify/backend";
import { DockerImage, Duration } from "aws-cdk-lib";
import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
import { execSync } from "child_process";
import * as path from "path";
import { fileURLToPath } from "url";

const functionDir = path.dirname(fileURLToPath(import.meta.url));

export const sendSmsGoFunctionHandler = defineFunction(
  (scope) =>
    new Function(scope, "SendSmsGo", {
      handler: "bootstrap",
      runtime: Runtime.PROVIDED_AL2023, // or any other python version
      timeout: Duration.seconds(3), //  default is 3 seconds
      code: Code.fromAsset(functionDir, {
        bundling: {
          image: DockerImage.fromRegistry("dummy"),
          local: {
            tryBundle(outputDir: string) {
              execSync(`rsync -rLv ${functionDir}/* ${path.join(outputDir)}`);
              execSync(
                `cd ${path.join(outputDir)} && GOARCH=amd64 GOOS=linux go build -tags lambda.norpc -o ${path.join(outputDir)}/bootstrap ${functionDir}/main.go`
              );
              return true;
            },
          },
        },
      }),
    }),
);
```